modules/ding_frontpage/ding_frontpage.pages_default.inc

```diff
@@ -170,7 +170,14 @@ function ding_frontpage_default_page_manager_pages() {
       'override_title_text' => '',
       'override_title_heading' => 'h2',
     );
-    $pane->cache = array();
+    $pane->cache = array(
+      'method' => 'lazy',
```

Comment:
I would think this required `lazy_pane` to be added as a dependency for `ding_frontpage`?